docs/configure-amazon-role.md

```diff
@@ -1,29 +1,38 @@
 # Configuring Amazon AWS
 
-The provisioning service uses Amazon's AWS IAM API to perform all provisioning tasks in order to achieve the best possible security standard and practices. In this article, we will describe how to create IAM User, Role and Policy and configure the application with it.
+The provisioning service uses Amazon's AWS IAM API to perform all provisioning tasks.
+We try to follow the best possible security standard and practices.
+In this article, we will describe how to create necessary IAM setup and configure this application with it.
 
 ## Configuring provisioning role
 
-Two accounts are needed for the setup:
+There are two accounts used in the provisioning flow:
 
-* **Service account** which is configured in the application (key/secret), this is an account of the service provider (e.g. Red Hat for cloud.redhat.com). It must not be root AWS account, the recommended practice is to create a dedicated user only with the minimum set of permissions.
-* **Tenant account** is the account into which the service account switches via AssumeRole API operation. It needs an AWS IAM Role with set of permissions to perform operations like list instance types, create new instance, start and stop. The role is identified by AWS ARN string which is fetched from the Sources microservice, Authorization endpoint.
+* **Service account** which is configured in the application (key/secret), this is an account of the service provider (e.g. Red Hat for cloud.redhat.com). 
+    It must not be root AWS account, the recommended practice is to create a dedicated user only with the minimum set of permissions.
+* **Tenant account** is the account into which the service account switches via AssumeRole API operation.
+    It needs an AWS IAM Role with set of permissions to perform operations this application is supposed to perform.
+    These are list instance types, create new instance, start and stop. The role is identified by AWS ARN string.
 
-You will need to know account numbers of both accounts. For development and testing purposes, both can be actually a single account.
+You will need to know account number of the Service account.
+For development and testing purposes, you can use the same account for both.
 
-### Service account policy
+### Service account
 
-Create a new policy for the Service account:
+This is the account that this application needs the credentials and will use to connect to AWS services.
 
-* Navigate to Identity and Access Management (IAM) on AWS.
-* Click on Policies - Create policy.
-* Click on JSON tab and paste the code below.
-* Click on your login name in the top-right corner and copy the Account ID to the clipboard. Then edit the "Resource" string and replace `123456789` with your account ID.
-* Click Next, Next.
-* Give the policy a name: `redhat-provisioning-service`
-* Click on Create Policy
-* Select **Access key - Programmatic access** for access type.
```

Comment:
removing this, I believe it is not relevant to Policy, right?

Reply:
Perhaps copy & paste error.